src/sentry/static/sentry/app/views/settings/incidentRules/triggers/actionsPanel/index.tsx

```diff
@@ -97,7 +101,19 @@ const getActionUniqueKey = ({
 const getFullActionTitle = ({
   type,
   integrationName,
-}: Pick<MetricActionTemplate, 'type' | 'integrationName'>) => {
+  sentryAppName,
+  status,
+}: Pick<
+  MetricActionTemplate,
+  'type' | 'integrationName' | 'sentryAppName' | 'status'
+>) => {
+  if (sentryAppName) {
+    if (status === 'internal') {
```

Comment:
@mgaeta We should do the same thing with unpublished for what it's worth

Reply:
I've updated it to always show that status. Are `"(internal)"` and `"(unpublished)"` clear enough?

Reply:
@mgaeta Yea, those are very clear. Thanks!